e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/endpointFocusAndPrefix/EndpointFocusAndPrefixTest.java

```diff
@@ -0,0 +1,453 @@
+package org.evomaster.e2etests.spring.examples.endpointFocusAndPrefix;
+
+import com.foo.rest.examples.spring.endpointFocusAndPrefix.EndpointFocusAndPrefixController;
+
+import org.evomaster.client.java.instrumentation.shared.ClassName;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.OpenApiAccess;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.Paths;
+import io.swagger.v3.oas.models.PathItem;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EndpointFocusAndPrefixTest extends SpringTestBase {
+
+    @BeforeAll
+    /*
+     */
+    public static void initClass() throws Exception {
+        SpringTestBase.initClass(new EndpointFocusAndPrefixController());
+    }
+
+    @Test
+    public void testRunBlackboxWithoutFocusOrPrefix() {
+
+        String outputFolder = "EndPointFocusAndPrefix";
+
+        List<String> args = getArgsWithCompilation(
```

Comment:
why not using `runTestHandlingFlakyAndCompilation()`?

Reply:
Hi Andrea, 

Thank you so much for your comment. I have fixed that for every test case.